### PR TITLE
backend/feat :created safety dashboard service

### PR DESCRIPTION
**Pull request by ashishsaini25**
## Type of Change

- [ ] Bugfix
- [x] New feature
- [ ] Enhancement
- [ ] Refactoring
- [ ] Dependency updates

## Description

Created safety dashboard service

### Additional Changes

- [ ] This PR modifies the database schema (database migration added)
- [ ] This PR modifies dhall configs/environment variables
- [ ] This PR contains API breaking changes

## Checklist

- [ ] I formatted the code and addressed linter errors `./dev/format-all-files.sh`
- [ ] I reviewed submitted code
- [ ] I added unit tests for my changes where possible
- [ ] I added a [CHANGELOG](/CHANGELOG.md) entry if applicable
- [ ] No leak detected in leakcanary

### CODE REVIEW

File: Backend/app/alchemist/spec/safety-dashboard/Storage/safetyDashboard.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  tableName: user
+
+  fields:
+    dl : Text
+    firstName : Text
+    lastName : Text
+    flaggedStaus : FlaggedStatus
+    flaggedCounter : Int
+    updatedAt : UTCTime
+    updatedAt : UTCTime
```
Comment (0utkarsh) [inline Backend/app/alchemist/spec/safety-dashboard/Storage/safetyDashboard.yaml:17]: `updatedAt` is twice

Comment (0utkarsh) [inline Backend/app/alchemist/spec/safety-dashboard/Storage/safetyDashboard.yaml:17]: Let's add `merchantId` and `merchantOperatingCityId` too?

File: Backend/app/alchemist/spec/safety-dashboard/Storage/safetyDashboard.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    FlaggedStatus:
+      enum: "Flagged,Enquiry,Guilty,Clean,Released"
+
+  queries:
+    findBydl:
+      kvFunction: findOneWithKV
+      where: dl
+
+
+UserFLaggedTable :
```
Comment (0utkarsh) [inline Backend/app/alchemist/spec/safety-dashboard/Storage/safetyDashboard.yaml:31]: Typo, `L` should be in lowercase

File: Backend/app/dashboard/Lib/src/Domain/Action/Dashboard/Merchant.hs [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+import Kernel.Types.Id
+import Kernel.Utils.Common
+import qualified Storage.Queries.Merchant as QMerchant
+import qualified Storage.Queries.MerchantAccess as QAccess
+import qualified Storage.Queries.MerchantAccess as QMerchantAccess
+import qualified Storage.Queries.Role as QRole
+import Tools.Auth
+import Tools.Auth.Merchant
+import Tools.Error
+
```
Comment (0utkarsh) [inline Backend/app/dashboard/Lib/src/Domain/Action/Dashboard/Merchant.hs:37]: These changes are in another PR?

File: Backend/app/alchemist/spec/safety-dashboard/API/safetyDashboard.yaml [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+apis:
+  # POST /upload/user
+  - POST:
+      endpoint: /upload/partner
+      params:
+
+      request:
+        type: API.Types.UI.Partner.PartnersReq
+      response:
+        type: API.Types.UI.Partner.PartnersResp
```
Comment (0utkarsh) [inline Backend/app/alchemist/spec/safety-dashboard/API/safetyDashboard.yaml:30]: Where are the files generated by these yamls?

**Review by 0utkarsh — CHANGES_REQUESTED** [review]

File: Backend/app/alchemist/spec/safety-dashboard/API/safetyDashboard.yaml (hunk 4 above)
Comment (AshishSaini25) [reply Backend/app/alchemist/spec/safety-dashboard/API/safetyDashboard.yaml:30]: this is incomplete pr , work in progress

**Review by khuzema786 — APPROVED** [review]

**Review by NikithShetty — CHANGES_REQUESTED** [review]

**Review by NikithShetty — CHANGES_REQUESTED** [review]

**Review by NikithShetty — APPROVED** [review]

**Review by NikithShetty — DISMISSED** [review]

**Pull request merged**